themes/edge.edx.org/lms/static/sass/partials/base/_certificates.scss

```diff
@@ -0,0 +1,27 @@
+// Certificate overrides for edge.edx.org
+
+.certificate-card {
+    &.mode-honor, &.mode-audit {
+        border-color: $honor-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/honor.png');
+        }
+    }
+
+    &.mode-verified {
+        border-color: $verified-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/verified.png');
+        }
+    }
+
+    &.mode-professional {
```

Comment:
Also, [lms/static/sass/features/_learner-profile.scss](https://github.com/edx/edx-platform/pull/15868/files#diff-13eecfdca02bd431df024514423fb75d) doesn't have professional.  Should it?

Reply:
I had been thinking not, since this is an edx.org concept, but since it is in the code it seems safest to add it too.